spec/doublyLinkedList.js

```diff
@@ -0,0 +1,226 @@
+      expect(myDoublyLinkedList.find("Azazel").data).to.eql("Azazel")
+    })
+    it('Returns -1 if no nodes contain provided value', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Beelzebub")
+      expect(myDoublyLinkedList.find("Choronzon")).to.eql(-1)
+    })
+    it('Returns the first node containing the provided value', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Beelzebub")
+      myDoublyLinkedList.insert("Azazel")
+      expect(myDoublyLinkedList.find("Azazel").next.next.data).to.eql("Azazel")
+      expect(myDoublyLinkedList.find("Azazel").next.data).to.eql("Beelzebub")
+    })
+  })
+
+  context('insertFirst()', () => {
+    it('Inserts new node at begining of list, which links to original' +
+      'head node', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insertFirst("Beelzebub")
+      expect(myDoublyLinkedList.getHeadNode().data).to.eql("Beelzebub")
+      expect(myDoublyLinkedList.getHeadNode().next.data).to.eql("Azazel")
+      expect(myDoublyLinkedList.getTailNode().previous.data).to.eql("Beelzebub")
+    })
+  })
+
+  context('insertAfter()', () => {
+    it('Inserts new node after the provided searchTerm', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Choronzon")
+      myDoublyLinkedList.insertAfter("Azazel", "Beelzebub")
+      expect(myDoublyLinkedList.find("Beelzebub").previous.data).to.eql("Azazel")
+      expect(myDoublyLinkedList.find("Beelzebub").next.data).to.eql("Choronzon")
+    })
+    it('returns -1 if the searchTerm does not exist in the list', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Choronzon")
+      expect(myDoublyLinkedList.insertAfter("Beelzebub", "Demogorgon")).to.eql(-1)
+    })
+  })
+
+  context('insertBefore()', () => {
+    it('Inserts new node before specified searchTerm', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Choronzon")
+      myDoublyLinkedList.insertBefore("Choronzon", "Beelzebub")
+      expect(myDoublyLinkedList.find("Beelzebub").next.data).to.eql("Choronzon")
+    })
+    it('returns -1 if the searchTerm does not exist in the list', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
+      myDoublyLinkedList.insert("Azazel")
+      myDoublyLinkedList.insert("Choronzon")
+      expect(myDoublyLinkedList.insertBefore("Beelzebub", "Demogorgon")).to.eql(-1)
+    })
+    it("Changes the next/prev values of the node previous to node with the" +
+      "searchTerm to be the value of the new node", () => {
```

Comment:
Are these two strings for the first argument of the function written in two lines?